tools/testsys-config/src/lib.rs

```diff
@@ -187,6 +191,8 @@ pub struct GenericVariantConfig {
     pub conformance_image: Option<String>,
     /// The custom registry used for conformance testing
     pub conformance_registry: Option<String>,
+    /// The control plane endpoint for vmware cluster creation
+    pub control_plane_endpoint: Option<String>,
```

Comment:
To clarify a bit more.
```suggestion
    /// The endpoint IP to reserve for the vSphere control plane VMs when creating a K8s cluster
    pub control_plane_endpoint: Option<String>,
```

Same comment applies to the parameter in `tools/testsys/src/run.rs`